doc/source/ray-overview/examples.rst

```diff
@@ -104,21 +104,21 @@ Ray Examples
         Aviary toolkit serving live traffic for LLMs
 
     .. grid-item-card:: :bdg-success:`Tutorial`
-        :class-item: gallery-item pytorch training
+        :class-item: gallery-item pytorch
         :link: /train/examples/pytorch/convert_existing_pytorch_code_to_ray_train
         :link-type: doc
 
         Get started with Ray Train from an existing PyTorch codebase
 
     .. grid-item-card:: :bdg-secondary:`Code example`
-        :class-item: gallery-item training
```

Comment:
Intuitively, the `training` tag on this is right, @matthewdeng is there a reason why we don't include it in the Train examples page?